Week5/Week5_Exercise2.js

```diff
@@ -0,0 +1,56 @@
+let taskList = [];
+
+function deleteItem(event) {
+    event.target.parentNode.remove();
+}
+
+function addTask(event) {
+    if (event.keyCode !== 13) {
```

Comment:
A comment to dispel any mystery surrounding magic numbers is necessary :heart:

Reply:
I was going to say something in class, but I went ahead and added a comment too. Magic numbers are a bad habit.